src/data/real/basic.lean

```diff
@@ -479,6 +479,13 @@ have bdd_above {x | -x ∈ s} → bdd_below s, from
 have ¬ bdd_above {x | -x ∈ s}, from mt this hs,
 neg_eq_zero.2 $ Sup_of_not_bdd_above $ this
 
+lemma zero_le_Inf (S : set ℝ) (hS : ∀ x ∈ S, (0:ℝ) ≤ x) : 0 ≤ Inf S :=
```

Comment:
`zero_le_...` lemmas are most often called `..._nonneg` instead. I think that the name of that lemma should be `Inf_nonneg`.
```suggestion
lemma Inf_nonneg (S : set ℝ) (hS : ∀ x ∈ S, (0:ℝ) ≤ x) : 0 ≤ Inf S :=
```